utils/storage.ts

```
@@ -0,0 +1,64 @@
+      const index = thoughts.findIndex(thought => thought.id === id);
+      
+      if (index !== -1) {
+        thoughts[index] = { ...thoughts[index], ...updates };
+        await AsyncStorage.setItem(STORAGE_KEY, JSON.stringify(thoughts));
+      }
+    } catch (error) {
+      console.error('Error updating thought:', error);
+      throw error;
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)